scripts/Muon/maxent_model.py

```diff
@@ -1,70 +1,122 @@
 from __future__ import (absolute_import, division, print_function)
+
 from six import iteritems
+
 import mantid.simpleapi as mantid
 
 
-class MaxEntModel(object):
-    # A simple class to hold the MaxEnt algorithm
+class MaxEntWrapper(object):
 
-    def __init__(self):
-        self.alg=mantid.AlgorithmManager.create("MuonMaxent")
-        self.alg.initialize()
-        self.alg.setChild(True)
+    """
+    A class to wrap the different parts
+    of the MaxEnt and its preprocessing.
+    This keeps the main MaxEnt class simple.
+    """
+
+    def __init__(self, maxent):
+        self.name = "MaxEnt"
+        self.model = maxent
 
-    def setInputs(self,inputs,runName):
-        self.inputs=inputs
-        for name,value in iteritems(self.inputs):
-            self.alg.setProperty(name,value)
-        self.run=runName
+    def loadData(self, inputs):
+        """
+        store the data in the wrapper for later
+        """
+        if "phaseTable" in inputs:
```

Comment:
You could also use `dict.get` to improve readability here:
```python
self.phaseTable = inputs.get("phaseTable", None)
```
returns the `phaseTable` if it exists otherwise it returns the second argument, in this case `None`. I think this is worth updating for the lines here.

Reply:
so we would have your line above all of the if statements and then the if statements would just be 

```python
x = inputs.get("phaseTable",None)
if x is not None:
```

Reply:
No problem.